pkg/crc/machine/machine.go

```diff
@@ -184,6 +184,18 @@ func Start(startConfig StartConfig) (StartResult, error) {
 		logging.WarnF("Failed Public dns query: %v : %s", err, queryOutput)
 	}
 
+	// Copy Kubeconfig file from bundle extract path to machine directory.
+	// In our case it would be ~/machine/crc
+	logging.InfoF("Copying kubeconfig file to instance dir ...")
+	if err := crcos.CopyFileContents(
+		filepath.Join(extractedPath, "kubeconfig"),
+		filepath.Join(constants.MachineInstanceDir, constants.DefaultName, "kubeconfig"),
```

Comment:
if we use DefaultName here, we assume that there will always be a single instance. We should be using MachineConfig.Name here.